.github/workflows/test-examples.yml

```diff
@@ -0,0 +1,17 @@
+name: Test deployment of examples to localstack
```

Comment:
I renamed this, because some of the `make test` tests also use localstack, so I didn't think calling these the "localstack tests" seemed very meaningful.